### PR TITLE
Change admin_stats dashboard to use cached values.

### PR DESCRIPTION
**Pull request by ashercodeorg**
These values are saved to the pegasus properties table by the admin_stats_cron.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Failing test should be unrelated:
CertificateImageTest
  test_image_generation                                          ERROR (0.15s)
NoMethodError:         NoMethodError: undefined method `[]' for nil:NilClass
            /home/ubuntu/code-dot-org/lib/cdo/graphics/certificate_image.rb:76:in`create_course_certificate_image'
            /home/ubuntu/code-dot-org/pegasus/test/test_certificate_image.rb:46:in `test_image_generation'

File: dashboard/app/views/admin_reports/admin_stats.html.haml [hunk 1]
```diff
@@ -1,88 +1,91 @@
 %h1 Admin stats
 
-%h3 Users with accounts
+-if @stats && @stats.length > 0
```
Comment (laurelfan) [inline dashboard/app/views/admin_reports/admin_stats.html.haml]: you can use `@stats.present?` here http://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark

**Comment by laurelfan** [comment]
LGTM -- I poked the rebuild button on Circle, maybe it was an intermittent failure?

File: dashboard/app/views/admin_reports/admin_stats.html.haml (hunk 1 above)
Comment (ashercodeorg) [inline dashboard/app/views/admin_reports/admin_stats.html.haml]: Done (nice!).

**Pull request merged**